packages/svelte/src/lib/overlay/styles/popover.css

```
@@ -1,4 +1,4 @@
-@reference '$style/index.css';
+@reference '@celar-ui/svelte/styles/index.css';
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@reference" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)